h2o-core/src/main/java/water/api/RapidsHandler.java

```diff
@@ -32,17 +32,8 @@ public RapidsSchemaV3 exec(int version, RapidsSchemaV3 rapids) {
       RapidsHandler.SESSIONS.put(rapids.session_id, ses);
     }
 
-    Val val;
```

Comment:
Keep as-is

Reply:
why? whats the point of re-throwing some errors and rethrowing and logging (twice) some other errors? functionally this code does nothing.

Reply:
reverted